docs/source/developers/crossbow.rst

```diff
@@ -0,0 +1,284 @@
+   integrations on for the newly created queue repository.
+
+   -  turn off Travis’ `auto cancellation`_ feature on branches
+
+3. Clone the newly created, by default the scripts looks for
+   ``crossbow`` next to arrow repository.
+
+   .. code:: bash
+
+      git clone https://github.com/<user>/crossbow crossbow
+
+   **Important note:** Crossbow only supports GitHub token based
+   authentication. Although it overwrites the repository urls provided with ssh
+   protocol, it's advisable to use the HTTPS repository URLs.
+
+4. `Create a Personal Access Token`_
+
+5. Locally export the token as an environment variable:
+
+   .. code:: bash
+
+      export CROSSBOW_GITHUB_TOKEN=<token>
+
+   ..
+
+      or pass as an argument to the CLI script ``--github-token``
+
+6. Export the previously created GitHub token on both CI services:
+
+   Use ``CROSSBOW_GITHUB_TOKEN`` encrypted environment variable. You can
+   set them at the following URLs, where ``ghuser`` is the GitHub
+   username and ``ghrepo`` is the GitHub repository name (typically
+   ``crossbow``):
+
+   -  TravisCI: ``https://travis-ci.org/<ghuser>/<ghrepo>/settings``
+   -  Appveyor:
+      ``https://ci.appveyor.com/project/<ghuser>/<ghrepo>/settings/environment``
+   -  CircleCI:
+      ``https://circleci.com/gh/<ghuser>/<ghrepo>/edit#env-vars``
+
+   On Appveyor check the ``skip branches without appveyor.yml`` checkbox
+   on the web UI under crossbow repository’s settings.
+
+7. Install Python 3.6:
+
+   Miniconda is preferred, see installation instructions:
+   https://conda.io/docs/user-guide/install/index.html
+
+8. Install the python dependencies for the script:
+
+   .. code:: bash
+
+      conda install -c conda-forge -y \
+          jinja2 \
+          pygit2 \
+          click \
+          ruamel.yaml \
+          setuptools_scm \
+          github3.py \
+          python-gnupg \
+          toolz \
+          jira
+
+   .. code:: bash
+
+      # pygit2 requires libgit2: http://www.pygit2.org/install.html
+      pip install \
+          jinja2 \
+          pygit2 \
+          click \
+          ruamel.yaml \
+          setuptools_scm \
+          github3.py \
+          python-gnupg \
+          toolz \
+          jira
+
+9. Try running it:
+
+   .. code:: bash
+
+      $ python crossbow.py --help
+
+Usage
+-----
+
+The script does the following:
+
+1. Detects the current repository, thus supports forks. The following
+   snippet will build kszucs’s fork instead of the upstream apache/arrow
+   repository.
+
+   .. code:: bash
+
+      $ git clone https://github.com/kszucs/arrow
+      $ git clone https://github.com/kszucs/crossbow
+
+      $ cd arrow/dev/tasks
+      $ python crossbow.py submit conda-win conda-linux conda-osx
+
+2. Gets the HEAD commit of the currently checked out branch and
+   generates the version number based on `setuptools_scm`_. So to build
+   a particular branch, just check out before running the script:
```

Comment:
Remove "just" since it's not "just" checking out, according to the "Note" that follows.

Reply:
And this "how to" should go somewhere else. This section IIUC is explaining what crossbow does when you call it, not how to control it.